_rules/image-not-in-acc-tree-is-decorative-e88epe.md

```diff
@@ -31,18 +31,18 @@ htmlHintIgnore:
 
 Any `img`, `canvas` or `svg` element that is [visible][] and is not [included in the accessibility tree][], except if one of the following is true:
 
-- **unloaded img**: the element is an `img` and it's [current request][]'s [state][image request state] is not [completely available][]; or
+- **unloaded img**: the element is an `img` and its [current request][]'s [state][image request state] is not [completely available][]; or
 - **ignored svg**: the element is an `svg` with an empty (`""`) [accessible name][] and a [semantic role][] of `graphics-document`; or
 - **ignored canvas**: the element is a `canvas` with an empty (`""`) [accessible name][] and no [explicit semantic role][]; or
 - **named from author**: the element is a [descendant][] in the [flat tree][] of an element that is [named from author][].
 
-**Note**: An example of an image ignored because of "named from author" is when it in a `button` element that uses `aria-label` for its accessible name.
+**Note**: An example of an image ignored because of "named from author" is when the image is a descendant of a `button` element that uses `aria-label` for its accessible name.
 
 ## Expectation
 
 Each test target is [purely decorative][].
 
-**note**: It is relatively common for an informative image such as an icon to be marked up as decorative, if the text alternative is adjacent to the image. This is a [conforming alternative version][] for the image. This fails the rule, but meets [conformance requirement 1 of WCAG 2.1](https://www.w3.org/TR/WCAG21/#cc1).
+**Note**: It is relatively common for an informative image such as an icon to be marked up as decorative, if the text alternative is adjacent to the image. This is a [conforming alternative version][] for the image. This fails the rule but meets [conformance requirement 1 of WCAG 2.1](https://www.w3.org/TR/WCAG21/#cc1).
```

Comment:
Shouldn't this note be moved after the second assumption? It looks like they are about the same kind of stuff (namely, assuming that there is no conforming alternative to the image)

Reply:
You are right it is related, but I think this is most relevant in the expectation. Would prefer to keep it here.